### PR TITLE
ensure_unlocked_desktop: Handle the case were gnome show a transient black screen

### PR DESCRIPTION
**Pull request by ggardet**
Fix https://openqa.opensuse.org/tests/741897#step/updates_packagekit_gpk/11

Related ticket: https://progress.opensuse.org/issues/39734

### CODE REVIEW

File: lib/utils.pm [hunk 1]
```diff
@@ -446,9 +446,14 @@ sub ensure_unlocked_desktop {
             last;            # desktop is unlocked, mission accomplished
         }
         if (match_has_tag 'screenlock') {
-            wait_screen_change {
-                send_key 'esc';    # end screenlock
-            };
+            unless (get_var('DESKTOP', '') =~ m/gnome/ && check_screen('blackscreen', 0)) {
```
Comment (rwx788) [inline lib/utils.pm]: check_screen has 0 timeout by default

**Comment by ggardet** [comment]
@rwx788 updated with your comments

**Comment by rwx788** [comment]
Sorry, just minor one, could you please squash commits?

**Comment by ggardet** [comment]
@rwx788 done

**Pull request merged**